Library/Formula/konoha.rb

```diff
@@ -14,14 +15,13 @@ class Konoha < Formula
   depends_on "json-c"
   depends_on "sqlite"
   depends_on "mecab" if MacOS.version >= :mountain_lion
-  depends_on :python # for python glue code
+  depends_on :python if MacOS.version <= :snow_leopard # for python glue code
 
   def install
-    args = std_cmake_args + [".."]
     mkdir "build" do
-      system "cmake", *args
+      system "cmake", "..", *std_cmake_args
       system "make"
-      system "make", "test" if build.include? "tests"
+      system "make", "test" if build.with?("tests") || build.bottle?
```

Comment:
In more general point, I think we should always enable test by default. The only exception I can think of is that test is buggy or take enormous long time and it's not security or crypto formula.